web/thesauruses/c/17/functions.json

```diff
@@ -0,0 +1,90 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "functions"
+    },
+    "concepts": {
+        "void_function_no_parameters": {
+            "code": "void func(){return;}",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and takes no parameters"
+        },
+        "void_function_with_parameters": {
+            "code": "void foo(char c, int i, double d) { return; }",
```

Comment:
Ideally it'd be nice to have this on multiple lines (either by adding `\n` in the string or by making "code" an array of lines) as typically people in C don't write functions all on one line.